src/main/java/de/tum/in/www1/artemis/domain/Course.java

```diff
@@ -102,6 +103,10 @@
     @Column(name = "presentation_score")
     private Integer presentationScore;
 
+    @Nullable
```

Comment:
Why is it nullable if false is the default value? That would imply it is never null? You could also set it using liquibase for all already existing courses to false.

Reply:
Oh yeah, I forgot about that. Removing it now.